src/php/utils/docker/docker-php-dev-mode

```diff
@@ -29,7 +29,15 @@ case "$1" in
 			exit 1
 		fi
 
-		pecl install xdebug-stable || pecl install xdebug-2.9.8
+		# If php version >= 8.0 install the right version of xdebug and linux headers
+		php_version=$(php -r 'echo PHP_VERSION;' | cut -c1-3 | tr -d '\n')
+		if [ "$(echo "$php_version >= 8.0" | bc)" = 1 ]; then
+			apk add --update linux-headers
+			pecl install xdebug-stable
```

Comment:
Shall we pin a version < 3.2.0 to avoid the issues we are experiencing?

Reply:
@carusogabriel the real problem today I think was the `linux-headers` [here ](https://gitlab.usabilla.net/usabilla/dev/account-management/-/jobs/2414206#L2662)

Reply:
so may I merge the PR?

Reply:
@ricardofontanelli That would still fail if we don't have those headers installed tho